WorkspaceServer/FileSystemDirectoryAccessor.cs

```diff
@@ -16,6 +17,10 @@ public class FileSystemDirectoryAccessor : IDirectoryAccessor
     {
         private readonly DirectoryInfo _rootDirectory;
 
+        public FileSystemDirectoryAccessor(string value): this(new DirectoryInfo(value))
```

Comment:
It would be good to rename `value` to something more informative.

Reply:
Done in https://github.com/dotnet/try/pull/380/commits/85257c62d3b42fd47d7ba43cec53ce16eb056f37